### PR TITLE
feat(core/canvas): return removed element

### PR DESCRIPTION
**Pull request by marstamm**
small improvement for the Canvas API

### CODE REVIEW

File: lib/core/Canvas.js [hunk 1]
```diff
@@ -581,7 +607,6 @@ Canvas.prototype.removeRootElement = function(rootElement) {
  * and returns the new root element.
  *
  * @param {Object|djs.model.Root} rootElement
- * @param {boolean} override
```
Comment (marstamm) [inline lib/core/Canvas.js:584]: Not sure if this is the way it should be done in JSDocs. I could not find information on how to mark it `@deprecated`

Comment (nikku) [reply lib/core/Canvas.js:584]: Well, removing it from public API is one way to do it. Every linter would complain though.

File: lib/core/Canvas.js [hunk 2]
```diff
@@ -524,6 +529,14 @@ Canvas.prototype.getRootElement = function() {
...
 };
 
+/**
+ * Adds a given root element and returns it.
+ *
+ * @param {Object|djs.model.Root} rootElement
+ *
+ * @return {Object|djs.model.Root} rootElement
+ */
+
```
Comment (nikku) [inline lib/core/Canvas.js:539]: Remove space before method.

File: lib/core/Canvas.js [hunk 3]
```diff
@@ -571,6 +595,8 @@ Canvas.prototype.removeRootElement = function(rootElement) {
   this._planes = this._planes.filter(function(plane) {
     return plane.rootElement !== rootElement;
   });
+
+  return rootElement;
```
Comment (nikku) [inline lib/core/Canvas.js:599]: :+1:

**Review by nikku — APPROVED** [review]

**Comment by nikku** [comment]
Sorry you cannot rely on this one with diagram-js@next yet :monkey_face:

**Pull request merged**